docs/WritingTests.asciidoc

```diff
@@ -79,6 +80,7 @@ sub test_flags {
     # 'ignore_failure' - if this module fails, it will not affect the overall result at all
     # 'milestone'      - after this test succeeds, update 'lastgood'
     # 'norollback'     - don't roll back to 'lastgood' snapshot if this fails
+    # 'force_reset'    - roll back to 'lastgood' snapshot even if test was successful (supported on QEMU backend only)
```

Comment:
I wonder if we shouldn't name 'QEMU backend' the default - sounds less scary :)

Reply:
Right, we don't mention it above as well. I will replace with something like "if supported by backend"

Reply:
It didn't bother me on the first review - but I just had the thought that force_reset sounds a lot like a boot of the SUT. In the light of 'norollback', how about always_rollback (and possibly renaming norollback to no_rollback).

Reply:
I also thought about different name, but grabbed one from the progress ticket. As we use norollback just in a single place, it's easy change. So +1 here.